microcosm-fastapi/microcosm_fastapi/context.py

```diff
@@ -0,0 +1,86 @@
+from contextlib import asynccontextmanager
+from functools import wraps
+
+from microcosm_postgres.operations import recreate_all
+from sqlalchemy.ext.asyncio import AsyncSession
```

Comment:
Does this file belong in this library, or in `microcosm-postgres`?

Reply:
Eventually it'll be in microcosm-postgres. For convenience right now I'm maintaining async Postgres and pubsub code within this repo. But the longterm homes are certainly better in their respective repos.